File: samples/client/3_0_3_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/IntegerTypeMatchesIntegers.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.IntJsonSchema;

public class IntegerTypeMatchesIntegers {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class IntegerTypeMatchesIntegers1 extends IntJsonSchema {
        private static @Nullable IntegerTypeMatchesIntegers1 instance = null;
        public static IntegerTypeMatchesIntegers1 getInstance() {
            if (instance == null) {
                instance = new IntegerTypeMatchesIntegers1();
            }
            return instance;
        }
    }

}

File: samples/client/3_0_3_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/IntegerTypeMatchesIntegers.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.IntJsonSchema

class IntegerTypeMatchesIntegers: IntJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class IntegerTypeMatchesIntegers1 : IntJsonSchema.IntJsonSchema1() {
        companion object {
            @Volatile
            private var instance: IntegerTypeMatchesIntegers1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: IntegerTypeMatchesIntegers1().also { instance = it }
                }
        }
    }

}
